learning_cp/knapsack/knapsack.jl

```diff
@@ -82,10 +84,11 @@ function trytrain(nbEpisodes::Int)
 
     #saving model weights
     trained_weights = params(approximator_model)
-    @save "model_weights_knapsack"*string(knapsack_generator.nb_items)*".bson" trained_weights
-
+    #@save "model_weights_knapsack"*string(knapsack_generator.nb_items)*".bson" trained_weights
+    
     return metricsArray, eval_metricsArray
 end
 
-
+CUDA.allowscalar(false)
```

Comment:
what does this line do ?

Reply:
@PierreTsr ?

Reply:
It makes sure that no iterative computation is done on a GPU. It prevents huge performance losses if a code in the nn-structure is poorly written (and will trigger an error)